.gitignore

```diff
@@ -1 +1,160 @@
-__pycache__
+# Byte-compiled / optimized / DLL files
+__pycache__/
+*.py[cod]
+*$py.class
+
+# C extensions
+*.so
+
+# Distribution / packaging
+.Python
+build/
+develop-eggs/
+dist/
+downloads/
+eggs/
```

Comment:
Most of the lines below don't look relevant, could they be removed?

Reply:
This is the standard gitignore from GitHub for Python projects. I would leave it

Reply:
I didn't realize it was that long, I'm good to leave it as is :)

Reply:
Great :) there's a whole repository of standardized gitignores, very useful